CHANGELOG.rst

```diff
@@ -14,6 +14,11 @@ Change Log
 Unreleased
 ~~~~~~~~~~
 
+[5.1.1] - 2024-01-31
```

Comment:
Remember to bump the version in code as well.

Reply:
Someone already updated the version here: https://github.com/openedx/edx-toggles/blob/c9ed5b02e39382266863abda35c95f81be3e2d39/edx_toggles/__init__.py#L5 
But it was never released: https://pypi.org/project/edx-toggles/

Reply:
Hah, OK. :-)